curriculum_learning.py

```
@@ -1417,6 +1428,39 @@ def stage5_ecg_cot(
+        """Stage 6: Chain-of-Thought Reasoning (CGM Diabetes Classification).
+
+        Configuration:
+        - Epochs: 60
+        - OpenTSLMSP: encoder_lr=2e-4, projector_lr=1e-4
+        - OpenTSLMFlamingo: base_lr=2e-4
+        - Metric: Test loss only
+        """
+
+        def cgm_accuracy(predictions, gold_answers):
+            correct = sum(
+                1 for pred, gold in zip(predictions, gold_answers)
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)